schema/annotation-source.yaml

```diff
@@ -89,6 +93,9 @@ $defs:
         description: MUST be "Agent".
       name:
         type: string
+      agent_type: 
```

Comment:
Figured this may be useful to be able to specify.  VA has considered defining specific Agent subtypes for each of these - but is waiting specific uses cases to arise. Thoughts?

Reply:
Okay by me.